pkg/config/connections.go

```diff
@@ -113,7 +110,20 @@ func writeConnectionConf(path string, conf *ConnectionsFile) error {
 // The function will read and write the file automatically and the
 // callback function just needs to modify the cfg as needed.
 func EditConnectionConfig(callback func(cfg *ConnectionsFile) error) error {
-	conf, path, err := readConnectionConf()
+	path, err := connectionsConfigFile()
+	if err != nil {
+		return err
+	}
+
+	lockPath := path + ".lock"
```

Comment:
Not a concern, more of a question: Do we want the lock file to be in a runtime directory location so the leftover files get cleaned up from a config location (vs state location)? I think we are in the clear on access, since if you can edit containers.conf you can likely also create files in the same dir.

Reply:
yeah, that is a fair point. I think because we only take the lock if we want to write the file it means we must have write access in the dir anyway so if this fails most likely the write would have been failed anyway.

Using a "rundir" is harder to maintain at least from within c/common, how do we know we can use the rundir? And what location should it be? Given we already established a working cross platform location I rather stick with that for safety reasons.

Reply:
agree, makes sense to me, simplicity is better on balance